scripts/ci.android.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -ex
 
 # Approve unapproved SDK licenses
-yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
```

Comment:
oh that is subtle! it has bitten me before as well
mostly commenting to say that after giving this a really focused attempt in #3055 I can really sympathize with the difficulty here. The RN59 build failure was confounding for me, this is really touchy to support a react-native version skew as great as RN59 to RN67

Reply:
Yes, I had to make this change for it to work on the build agent with updated sdk. I think it's 99% of the way to a working pr, but there's still a couple of small issues remaining. Thanks for your reviews and feedback on this!